internal/test/framework/framework.go

```diff
@@ -215,6 +220,16 @@ func (f *TestFramework) getOpenShiftOperatorClient(config *restclient.Config) er
 	return nil
 }
 
+// getK8sVersion gets the Kubernetes version in vx.x.x format
+func (f *TestFramework) getK8sVersion() error {
```

Comment:
There is overlapping functionality here with the getClusterVersion(). I would merge these two functions as they both have the purpose of getting the cluster version.

Reply:
Done